plugins/modules/idrac_secure_boot.py

```diff
@@ -0,0 +1,373 @@
+  platform_key:
+    type: path
+    description:
+      - Platform Key policy certificate path for UEFI Secure Boot.
+      - Absolute path of the certificate file I(import_certificates) is C(true).
+  key_exchange_key:
+    type: list
+    elements: path
+    description:
+      - Key exchange key policy certificate paths for UEFI Secure Boot.
+      - Absolute path of the certificate file if I(import_certificates) is C(import).
+  database:
+    type: list
+    elements: path
+    description:
+      - Databases certificate paths for UEFI Secure Boot.
+      - Absolute path of the certificate file if I(import_certificates) is C(import).
+  disallow_database:
+    type: list
+    elements: path
+    description:
+      - Disallow database certificate paths for UEFI Secure Boot.
+      - Absolute path of the certificate file if I(import_certificates) is C(import).
+  restart:
+    type: bool
+    default: false
+    description:
+      -  Restart the server to apply the secure boot settings.
+      - I(restart) will be ignored only when I(export_certificates) is C(true).
+  restart_type:
+    type: str
+    default: GracefulRestart
+    choices: [GracefulRestart, ForceRestart]
+    description:
+      - Reset type
+      - C(ForceRestart) Forcefully reboot the host system.
+      - C(GracefulRestart) Gracefully reboot the host system.
+      - C(GracefulRestart) Gracefully reboot the host system.
+      - I(restart_type) is applicable when i(restart) is C(true).
+  job_wait:
+    type: bool
+    default: true
+    description:
+      - Provides the option to wait for job completion.
+  job_wait_timeout:
+    type: int
+    default: 1200
+    description:
+      - The maximum wait time of I(job_wait) in seconds. The job is tracked only for this duration.
+      - This option is applicable when I(job_wait) is C(true).
+requirements:
+    - "python >= 3.9.6"
+author:
+    - "Abhishek Sinha(@ABHISHEK-SINHA10)"
+notes:
+    - This module reports changes found when ran in check_mode for I(import_certificates).
+    - This module does not support idempotency when I(import_certificates) is provided.
+    - This module supports both IPv4 and IPv6 address.
+    - This module supports C(check_mode).
+"""
+
+EXAMPLES = """
+---
+- name: Import a SecureBoot certificate.
+  dellemc.openmanage.idrac_secure_boot:
+    import_certificates: true
+    platform_key: /user/name/certificates/pk.pem
+    key_exchange_key:
+      - /user/name/certificates/kek1.pem
+      - /user/name/certificates/kek2.pem
+    database:
+      - /user/name/certificates/db1.pem
+      - /user/name/certificates/db2.pem
+    disallow_database:
+      - /user/name/certificates/dbx1.pem
+      - /user/name/certificates/dbx2.pem
+"""
+
+RETURN = r'''
+---
+msg:
+  description: Status of the secure boot operation.
+  returned: always
+  type: str
+  sample: "The Secure Boot Certificate Import operation has completed successfully."
```

Comment:
Suggestion:
Successfully imported the SecureBoot certificate.

Reply:
will discuss within team.

Reply:
sure will change it.